lib/github_core.ml

```diff
@@ -785,17 +789,23 @@ module Make(Env : Github_s.Env)(Time : Github_s.Time)(CL : Cohttp_lwt.Client)
     (* Convert a request body into a stream *)
     let realize_body = function None -> None | Some b -> Some (CLB.of_string b)
 
+    module Media_type =
+    struct
+      let v3 = "application/vnd.github.v3+json"
+      let experimental = "application/vnd.github.mockingbird-preview"
+    end
```

Comment:
I'm not sure this is necessary as most API requests will use the normal `v3` label and the experimental labels are specific to experimental features.